docs/08_ruby_style.md

```diff
@@ -1,10 +1,9 @@
 ## Ruby Style
 * [Rubocop](https://github.com/bbatsov/rubocop)
```

Comment:
Maybe we take out any references to Rubocop here? It seems that if they're looking for guidance and we give it to them, they don't need to know about the history of cookstyle, they just use it first because it's in the ChefDK.